keps/sig-storage/20190204-scale-testing.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Volume Scale Testing Plan
+authors:
+  - "@msau42"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-scalability
+reviewers:
+  - "@pohly"
+  - "@saad-ali"
+  - "@wojtek-t"
+approvers:
+  - "@saad-ali"
+  - "@wojtek-t"
+editor: TBD
+creation-date: 2019-02-04
+last-updated: 2019-02-04
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Volume Scale Testing Plan
```

Comment:
I would change to Scale and Performance.
Some of them don't really need to be run at scale.